### ISSUE TITLE
Build failed in example app

### ISSUE THREAD

**Issue by nikiJava** [ZaraclaJ/audio_recorder#49]
```
Launching lib/main.dart on Android SDK built for x86 in debug mode...
[!] Your app isn't using AndroidX.
    To avoid potential build failures, you can quickly migrate your app by following the steps on https://goo.gl/CP92wY.
Running Gradle task 'assembleDebug'...

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:transformResourcesWithMergeJavaResForDebug'.
> More than one file was found with OS independent path 'META-INF/proguard/androidx-annotations.pro'

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 9s
AndroidX incompatibilities may have caused this build to fail. Please migrate your app to AndroidX. See https://goo.gl/CP92wY.
Finished with error: Gradle task assembleDebug failed with exit code 1
```

It can be fixed with the following code in build.gradle inside `android {}`:
`    packagingOptions {
        exclude 'META-INF/proguard/androidx-annotations.pro'
    }`

**Comment by nikiJava** [comment]
https://github.com/ZaraclaJ/audio_recorder/pull/50

**Comment by ZaraclaJ** [comment]
Fix in version 1.0.2

**Comment by ened** [comment]
AndroidX support updated in #57
